Description: Remove badges from README.md
 They require internet to be loaded
Author: Andrea Pappacoda <andrea@pappacoda.it>
Forwarded: not-needed
Last-Update: 2022-11-06

--- zydis-4.0.0-rc.1.orig/README.md
+++ zydis-4.0.0-rc.1/README.md
@@ -1,15 +1,3 @@
-<p align="center">
-  <img alt="zydis logo" src="https://zydis.re/img/logo.svg" width="400px">
-</p>
-
-<p align="center">
-  <img src="https://img.shields.io/badge/License-MIT-blue.svg" alt="License: MIT">
-  <a href="https://github.com/zyantific/zydis/actions"><img src="https://github.com/zyantific/zydis/workflows/CI/badge.svg" alt="GitHub Actions"></a>
-  <a href="https://bugs.chromium.org/p/oss-fuzz/issues/list?sort=-opened&can=1&q=proj:zydis"><img src="https://oss-fuzz-build-logs.storage.googleapis.com/badges/zydis.svg" alt="Fuzzing Status"></a>
-  <a href="https://gitter.im/zyantific/zydis?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=body_badge"><img src="https://badges.gitter.im/zyantific/zyan-disassembler-engine.svg" alt="Gitter"></a>
-  <a href="https://discord.zyantific.com/"><img src="https://img.shields.io/discord/390136917779415060.svg?logo=discord&label=Discord" alt="Discord"></a>
-</p>
-
 <p align="center">Fast and lightweight x86/x86-64 disassembler and code generation library.</p>
 
 ## Features
